index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>comproDLS Assignment</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="main-container">
+    <div class="info-container">
+      <div class="header">
+        <div class="upper-heading">
+          comproDLS<sup>TM</sup>
+        </div>
+        <div class="main-heading">
+          OnBoarding
+          <hr>
+        </div>
+        <div class="sub-heading">
+          <p>Application</p>
+        </div>
+      </div>
+      <div class="content">
+        <p class="upper-text">Welcome to the comproDLS<sup>TM</sup> OnBoarding
+          Application for mentors and freshers. Please sign
+          in using your organisation email and password.
+        </p><br>
+        <p class="bottom-text">
+          In case you do not have an account, click on "Sign up
+          now" to get started.
+        </p>
+      </div>
+      <div class="small-form">
+
+        <div class="form">
+          <div class="form-heading">
+            Sign in
+          </div>
+          <form action="#">
+            <!-- <label for="emailInput">Email</label> -->
```

Comment:
If commented code is not needed, then you can remove it.

Reply:
removed it